cli/targets/static.js

```diff
@@ -394,8 +449,19 @@ function buildType(ref, type) {
         if (config.comments) {
             push("");
             var jsType = toJsType(field);
-            if (field.optional && !field.map && !field.repeated && (field.resolvedType instanceof Type || config["null-defaults"]) || field.partOf)
-                jsType = jsType + "|null|undefined";
+
+            // New behaviour - fields explicitly marked as optional and members of a one-of are nullable
```

Comment:
New/old behavior is vague (there will be a lot of changes coming!) Can you specify syntax, etc., in comments instead?

Reply:
After thinking about this comment, I changed the guard flag to --null-semantics. The inline comments are updated accordingly and changes pushed. See notes below.